### PR TITLE
Lum 29800 cycle lit images

### PR DESCRIPTION
**Pull request by Blackbaud-JohnNelson**
Make Ethan a .jpg rather than .gif
Add a few new lit images
Make it so we get a random lit image every 15 minutes.
 
@blackbaud/micro-cervezas 
@Blackbaud-RajRaulji 
@Blackbaud-GregHenry 
@Blackbaud-BethRichardson

### CODE REVIEW

**Comment by Blackbaud-JohnNelson** [comment]
Please wait for @Blackbaud-RajRaulji approval before merging

File: src/app/sprint-lit/sprint-lit.component.spec.ts [hunk 1]
```diff
@@ -21,5 +27,6 @@ describe('SprintLitComponent', () => {
 
   it('should create', () => {
     expect(component).toBeTruthy();
+    expect(component.litImage).not.toBeUndefined();
```
Comment (Blackbaud-DiHuynh) [inline src/app/sprint-lit/sprint-lit.component.spec.ts]: can we remove the double negative?

File: src/app/sprint-lit/sprint-lit.component.ts [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
 })
+
 export class SprintLitComponent {
+
+  litImage: string;
+  litImages: Array<string> = [
+    "../assets/ethanLIT.jpg",
+    "../assets/carolineLIT.jpg",
+    "../assets/gregLIT.jpg",
+    "../assets/rajLIT.jpg"
```
Comment (Blackbaud-DiHuynh) [inline src/app/sprint-lit/sprint-lit.component.ts]: maybe we can have a lit folder :D, and maybe this thing can do `"../assets/lit/*`

Comment (Blackbaud-JohnNelson) [reply src/app/sprint-lit/sprint-lit.component.ts]: @Blackbaud-DiHuynh I moved them to a lit directory, but did not do the second part. I want to play around with making lit image choice configurable in a separate PR.

File: src/app/sprint-lit/sprint-lit.component.spec.ts (hunk 1 above)
Comment (Blackbaud-JohnNelson) [reply src/app/sprint-lit/sprint-lit.component.spec.ts]: Good catch. Thank you.

**Pull request merged**